gen3-integration-tests/services/indexd.py

```diff
@@ -92,3 +93,32 @@ def delete_files(self, guids: list, user="indexing_account"):
                 index.delete_record(guid=guid)
             except Exception as e:
                 logger.exception(msg=f"Failed to delete record with guid {guid} : {e}")
+
+    def file_equals(self, res: dict, file_node: dict) -> None:
+        logger.info(f"Response data : {res}")
+        logger.info(f"File Node with CCs : {file_node.props}")
```

Comment:
```suggestion
    def file_equals(self, res: dict, file_record: dict) -> None:
        logger.info(f"Response data: {res}")
        logger.info(f"File record: {file_record.props}")
```
removing "with cc" because the record doesn't necessarily have consent codes here